modules/service/src/main/scala/lucuma/odb/service/ItcService.scala

```diff
@@ -388,7 +389,7 @@ object ItcService {
         client.imaging(ii, useCache = false)
           .map(_.result)
           .recover {
-            case ResponseException(errors, _) if errors.exists(_.message.contains("target is too bright")) =>
+            case ResponseException(errors, _) if errors.exists(_.extensions.exists(_.exists(_ === ("errorCode" -> ErrorCode.SourceTooBright.asJson)))) =>
```

Comment:
If there were other errors, besides the source-too-bright one, would we still want to use the default integration time?

Reply:
Good question. I'd say only this one, if e.g. the config is unsupported you ddon't really want to default to a valid value